docs/antora/modules/ROOT/pages/api_reference.adoc

```diff
@@ -23,24 +23,8 @@ Package v1alpha1 contains API Schema definitions for the cloud.redhat.com v1alph
 
 
 
-[id="{anchor_prefix}-github-com-redhatinsights-clowder-apis-cloud-redhat-com-v1alpha1-apipath"]
-==== APIPath
-
-_Underlying type:_ _string_
```

Comment:
Things look good but I think some of these api doc formatting changes are related to version mismatches on some of the generation tools. Try this:

1.  `rm -fr docs/build/crd-ref-docs`
2. Make sure you're using asciidoctor v2.0.20 (`asciidoctor -v`) if not, `sudo dnf update -y asciidoctor` or similar
3. `make pre-push` (crd-ref-docs will be downloaded and installed again at this point)


The doc changes to `apiPaths` we are seeing generated here will go in via #888